### PR TITLE
feat(Jira Server): Voting to different fields in Sprint Poker is now supported

### PR DESCRIPTION
**Pull request by Dschoordsch**
Resolves:
* #5812 

## Testing steps
* [ ] integrate with Jira Server and add different issue types to Sprint Poker
* [ ] different issue types have different available fields (i.e. Story has "Story Points" which is absent from Task in our test repo)
* [ ] verify the vote field is stored per issue type
* [ ] vote for the issues and verify the result

https://www.loom.com/share/4d45bafbd4d141008e724bebf9c0a230

### CODE REVIEW

**Review by igorlesnenko — COMMENTED** [review]
image
Should we create some default fields like we do for `jira` or maybe create a separate issue about this?

**Comment by Dschoordsch** [comment]
@igorlesnenko could you please take a look?

File: packages/server/postgres/migrations/1652081624156_addJiraServerDimensionFieldMap.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  CREATE TABLE IF NOT EXISTS "JiraServerDimensionFieldMap" (
+    "id" SERIAL UNIQUE,
+    "providerId" INTEGER NOT NULL,
```
[the remark this answers is not in the document]
Comment (Dschoordsch) [reply packages/server/postgres/migrations/1652081624156_addJiraServerDimensionFieldMap.ts:10]: I think we could, but that requires changes in other places as we would need to cascade delete providers. I will create a new issue as it's also not a foreign key for other integrations. #6564

**Comment by Dschoordsch** [comment]
I think we should not mess with the Jira configuration from Parabol. It is fairly easy to add fields manually (see loom)

**Review by igorlesnenko — APPROVED** [review]

**Pull request merged**